### ISSUE TITLE
Migrating/Cloning Workspaces

### ISSUE THREAD

**Issue by Kcorb95** [a85/POSTMan-Chrome-Extension#10494]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request

### Is your feature request related to a problem?

After setting up a workspace and members creating forks (environment files with their own settings), there does not seem to be a way to migrate or clone the entire workspace including environment files to a different account with a paid license.

It would appear that most things can be transferred if I import as a GitHub repo, but this doesn't take care of forks. If there was an easy way to do this at the workspace level as a whole, it would reduce friction in switching from a free to paid licenses.

### Describe the solution you'd like

An easy way to migrate or clone a workspace on a different account without having to fork each collection etc. individually.

**Comment by akshay-kulkarni-03** [comment]
Hi @chrisdeso thanks for opening this feature request. We are discussing the feasibility, usability of this feature internally. Will update this thread once I have more information on this.

**Comment by calebruz** [comment]
Yes. Please. I recently attempted to clone via postman API, which worked, but from what I was told is that those collections remain attached to the original workspace. This means that deleting the new workspace also emptied the original workspace. There was no documentation about this outcome that I could find. 

This is also an issue with moving collections between workspaces. There should be an option to move or copy. Currently, if you want to move a copy to a different workspace you have to duplicate and move which could be reduced to just adding a checkbox to the move modal asking if the user wants to move a copy vs the collection itself.

**Comment by MincDev** [comment]
Any feedback on this?

**Comment by LockTar** [comment]
Cloning a workspace would be very handy! Inside the same account is voor me personally enough.
Just ask for the new name or let people just select the target workspace (so you would first need to create an empty one).

**Comment by davidbrenner-insight** [comment]
My company has recently upgraded our postman licenses. I'd like to move my workspaces from my personal system to the team environment without having to export/import everything piece by piece. One export and one import for the whole workspace would be rather convenient.

**Comment by JaggedJax** [comment]
I'm excited to waste my day doing this. Let's just say it wasn't my decision and I would have just taken the opportunity to move elsewhere.

**Comment by becky-intelletive** [comment]
I am at a company that was recently acquired, so my old email is going away. I need a way to move my workspaces to my new email. One collection at a time is tedious :/

**Comment by natdevaccount** [comment]
@becky-intelletive You can just update the account email to the new one or use the data dump feature to export all of your collections and environments.
